src/Apps/Order2/Routes/Checkout/Components/Order2FulfillmentDetailsStep/Order2FulfillmentDetailsStep.tsx

```diff
@@ -0,0 +1,116 @@
+import { Box, Flex, Tab, Tabs, Text } from "@artsy/palette"
+import { Order2DeliveryForm } from "Apps/Order2/Routes/Checkout/Components/Order2FulfillmentDetailsStep/Order2DeliveryForm"
+import { Order2FulfillmentDetailsCompletedView } from "Apps/Order2/Routes/Checkout/Components/Order2FulfillmentDetailsStep/Order2FulfillmentDetailsCompletedView"
+import { Order2PickupForm } from "Apps/Order2/Routes/Checkout/Components/Order2FulfillmentDetailsStep/Order2PickupForm"
+import type { Order2FulfillmentDetailsStep_order$key } from "__generated__/Order2FulfillmentDetailsStep_order.graphql"
+import { useEffect, useState } from "react"
+import { graphql, useFragment } from "react-relay"
+
+interface Order2FulfillmentDetailsStepProps {
+  order: Order2FulfillmentDetailsStep_order$key
+}
+
+enum TempStepStatus {
+  ACTIVE = "ACTIVE",
+  COMPLETE = "COMPLETE",
+}
```

Comment:
Do we have the behavior specified in the design for collapsed/expanded state across sections? Like can user expand all sections, or does expanding one section automatically collapse others? How about unsaved changes?

Reply:
Separately, is this a `temp`name?

Reply:
This is temporary - I'm working on a higher-level context PR to build off this right now.